sdk/openai/openai-assistants/review/openai-assistants.api.md

```diff
@@ -0,0 +1,644 @@
+export interface ListMessagesOptions extends OperationOptions {
+    after?: string;
+    before?: string;
+    limit?: number;
+    order?: ListSortOrder;
+}
+
+// @public
+export interface ListResponseOf<T> {
+    data: T[];
+    firstId: string;
+    hasMore: boolean;
+    lastId: string;
+}
+
+// @public
+export interface ListRunsOptions extends OperationOptions {
+    after?: string;
+    before?: string;
+    limit?: number;
+    order?: ListSortOrder;
+}
+
+// @public
+export interface ListRunStepsOptions extends OperationOptions {
+    after?: string;
+    before?: string;
+    limit?: number;
+    order?: ListSortOrder;
+}
+
+// @public
+export type ListSortOrder = string;
+
+// @public
+export type MessageContent = MessageTextContent | MessageImageFileContent;
+
+// @public
+export interface MessageFile {
+    createdAt: Date;
+    id: string;
+    messageId: string;
+}
+
+// @public
+export interface MessageImageFileContent {
+    imageFile: MessageImageFileDetails;
+    type: "image_file";
+}
+
+// @public
+export interface MessageImageFileDetails {
+    fileId: string;
+}
+
+// @public
+export type MessageRole = string;
+
+// @public
+export type MessageTextAnnotation = MessageTextFileCitationAnnotation | MessageTextFilePathAnnotation;
+
+// @public
+export interface MessageTextContent {
+    text: MessageTextDetails;
+    type: "text";
+}
+
+// @public
+export interface MessageTextDetails {
+    annotations: MessageTextAnnotation[];
+    value: string;
+}
+
+// @public
+export interface MessageTextFileCitationAnnotation {
+    endIndex: number;
+    fileCitation: MessageTextFileCitationDetails;
+    startIndex: number;
+    text: string;
+    type: "file_citation";
+}
+
+// @public
+export interface MessageTextFileCitationDetails {
+    fileId: string;
+    quote: string;
+}
+
+// @public
+export interface MessageTextFilePathAnnotation {
+    endIndex: number;
+    filePath: MessageTextFilePathDetails;
+    startIndex: number;
+    text: string;
+    type: "file_path";
+}
+
+// @public
+export interface MessageTextFilePathDetails {
+    fileId: string;
+}
+
+// @public
+export class OpenAIKeyCredential implements KeyCredential {
+    constructor(key: string);
+    get key(): string;
+    update(newKey: string): void;
+}
+
+// @public
+export interface RequiredAction {
+    submitToolOutputs?: SubmitToolOutputsDetails;
+    type: string;
+}
+
+// @public
+export interface RequiredFunctionToolCall {
+    function: FunctionDefinition;
+    id: string;
+    type: "function";
+}
+
+// @public
+export type RequiredToolCall = RequiredFunctionToolCall;
+
+// @public
+export interface RetrievalToolCall {
+    id: string;
+    retrieval: Record<string, string>;
+    type: "retrieval";
+}
+
+// @public
+export interface RetrievalToolDefinition {
+    type: "retrieval";
+}
+
+// @public
+export interface RunError {
+    code: string;
+    message: string;
+}
+
+// @public
+export interface RunError {
+    code: string;
+    message: string;
+}
```

Comment:
Do you know why is this duplicated? Is it defined twice?

Reply:
This propagated from the generated, so the TypeSpec may have it twice. Will check next time I regen. Fixed for now.